pkg/controller/certificates/requestmanager/util_test.go

```diff
@@ -102,20 +101,16 @@ func createCryptoBundle(originalCert *cmapi.Certificate) (*cryptoBundle, error)
 	for k, v := range crt.Annotations {
 		annotations[k] = v
 	}
-	if crt.Status.Revision != nil {
-		annotations[cmapi.CertificateRequestRevisionAnnotationKey] = fmt.Sprintf("%d", crt.Status.Revision)
-	} else {
-		annotations[cmapi.CertificateRequestRevisionAnnotationKey] = "1"
-	}
 
+	annotations[cmapi.CertificateRequestRevisionAnnotationKey] = "NOT SET"
 	annotations[cmapi.CertificateRequestPrivateKeyAnnotationKey] = crt.Spec.SecretName
 	annotations[cmapi.CertificateNameKey] = crt.Name
 	if crt.Status.NextPrivateKeySecretName != nil {
 		annotations[cmapi.CertificateRequestPrivateKeyAnnotationKey] = *crt.Status.NextPrivateKeySecretName
 	}
 	certificateRequest := &cmapi.CertificateRequest{
 		ObjectMeta: metav1.ObjectMeta{
-			GenerateName:    crt.Name + "-",
```

Comment:
question: just checking my understanding quickly without diving in

Is this changed because the name will never be used?

Reply:
Yes, the CertificateRequest now have a Name instead of a GenerateName.
For each test, I updated the test and explicitly set the Name instead of relying on the "default" name.

Reply:
The name is "<certificate name>-<generation>"